package.json

```diff
@@ -1,12 +1,7 @@
 {
   "name": "arachne-ui-components",
-  "version": "1.13.39",
-  "licenses": [
-    {
-      "type": "Apache-2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0"
-    }
-  ],
+  "version": "1.15.2",
```

Comment:
This version has already been published: https://www.npmjs.com/package/arachne-ui-components